about.html

```diff
@@ -0,0 +1,176 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
```

Comment:
Great job on this part just fix the header size it is too small right now!

![Screenshot from 2023-08-02 15-15-50](https://github.com/memicelis/web-development-bootcamp/assets/118063058/57fa4ec3-e7da-4480-b1c4-540f12baeac0)
![Screenshot from 2023-08-02 15-15-56](https://github.com/memicelis/web-development-bootcamp/assets/118063058/b6b6541b-8ee3-48f7-a14e-b907c10316da)